lib/plugins/scoreboard.js

```diff
@@ -57,6 +58,47 @@ function inject (bot) {
     }
   })
 
+  bot.on('teams', (packet) => {
+    if (packet.mode === 0) {
+      const team = {}
+      team.displayName = packet.name
+      team.prefix = packet.prefix
+      team.suffix = packet.suffix
+      team.friendlyFire = packet.friendlyFire
+      team.nameTagVisibility = packet.nameTagVisibility
+      team.collisionRule = packet.collisionRule
+      team.color = packet.color
+      team.players = packet.players
```

Comment:
I'd map this array to the values of `bot.players` or add `bot.onlinePlayers` which does that

Reply:
im not sure what you mean by that, do you mean you want to instead of store a list of players in the team object, store the player's team in each of the team's players?

Reply:
I meant `team.onlinePlayers` which would map `team.players` to `bot.players` values. `player.team` is also a good idea